### PR TITLE
certify: Add show-cert command for debugging

### PR DESCRIPTION
**Pull request by kelunik**
As mentioned earlier, here's the `show-cert` command that can be used to view certificates.

### CODE REVIEW

File: tools/certify/commands/show-cert.cpp [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+                        std::cerr << "Invalid service specific permissions for CA" << std::endl;
+                        continue;
+                    }
+
+                    // See final draft ETSI EN 302 637-2 V1.3.1 (2014-09)
+                    if (ssp[0] == 0) {
+                        if (ssp.size() != 1) {
+                            std::cout << " - Warning: ssp.size() is expected to be 1, but was " << ssp.size() << std::endl;
+                        } else {
+                            std::cout << " - No version, shell be used only for testing." << std::endl;
```
Comment (riebl) [inline tools/certify/commands/show-cert.cpp]: "shall"

File: tools/certify/commands/show-cert.cpp [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+                        }
+                    } else if (ssp[0] == 1) {
+                        if (ssp.size() != 3) {
+                            std::cout << " - Warning: ssp.size() is expected to be 3, but was " << ssp.size() << std::endl;
+                        } else {
+                            if (ssp[1] & (1 << 7)) {
```
Comment (riebl) [inline tools/certify/commands/show-cert.cpp]: I think all this SSP decoding belongs to vanetza::security.

File: tools/certify/commands/show-cert.cpp [hunk 3]
```diff
@@ -0,0 +1,287 @@
+#include "keyio.hpp"
+#include "show-cert.hpp"
+#include <boost/algorithm/hex.hpp>
+#include <boost/date_time/posix_time/posix_time.hpp>
+#include <boost/program_options.hpp>
+#include <boost/variant.hpp>
```
Comment (riebl) [inline tools/certify/commands/show-cert.cpp]: not needed?

File: tools/certify/commands/sign-ticket.cpp [hunk 4]
```diff
@@ -113,8 +128,8 @@ int SignTicketCommand::execute()
     std::ofstream dest;
     dest.open(output.c_str(), std::ios::out | std::ios::binary);
 
-    vanetza::OutputArchive archive(dest);
-    vs::serialize(archive, certificate);
+    OutputArchive archive(dest);
+    serialize(archive, certificate);
```
Comment (riebl) [inline tools/certify/commands/sign-ticket.cpp]: All changes in sign-ticket.cpp do not belong the added "show-cert" feature, so these should be moved to a separate commit.

File: tools/certify/commands/show-cert.cpp (hunk 3 above)
Comment (kelunik) [reply tools/certify/commands/show-cert.cpp]: Removed.

File: tools/certify/commands/show-cert.cpp (hunk 1 above)
Comment (kelunik) [reply tools/certify/commands/show-cert.cpp]: Fixed.

File: tools/certify/commands/show-cert.cpp (hunk 2 above)
Comment (kelunik) [reply tools/certify/commands/show-cert.cpp]: Eventually, yes, but I'm not sure which names to use for the values of an enum.

File: tools/certify/commands/sign-ticket.cpp (hunk 4 above)
Comment (kelunik) [reply tools/certify/commands/sign-ticket.cpp]: Indeed, I shouldn't be lazy. Extracted to #26.

File: tools/certify/commands/show-cert.cpp (hunk 3 above)
Comment (kelunik) [reply tools/certify/commands/show-cert.cpp]: Reverted, it's actually needed because of `boost::get()`.

**Comment by riebl** [comment]
Merged 63d884992664f720a5533f7b052699249c9c3dc4.
Created ticket #28 for moving SSP code to vanetza::security.
